PED/SetPedCombatRange.md

```diff
@@ -17,5 +17,6 @@ Only the values 0, 1 and 2 occur in the decompiled scripts. Most likely refers d
 
 ## Parameters
 * **ped**: 
-* **p1**: 
+* **combat range**: 0, 1 or 2
```

Comment:
Parameter names cannot have spaces, they also need to match the declared parameters on top.

Reply:
yeah, this could probably just be `range` instead.
and an explanation of the variable would probably be beneficial